ci/.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: RSpec CI
+on: [pull_request, push]
```

Comment:
I believe this needs to be tweaked so we only run `push` on `main`/`*-maintenance`/`*-dev`

Reply:
Yeah agreed.

Reply:
Something like? I am not sur for pull_request.

```yml
on:
  # Trigger the workflow on push or pull request,
  # but only for the main branch
  push:
    branches:
      - main
      - *-maintenance
      - *-dev
  pull_request:
      branches:
        - **
```